src/webgpu/api/operation/rendering/indirect_draw.spec.ts

```diff
@@ -1,16 +1,10 @@
 export const description = `
 Tests for the indirect-specific aspects of drawIndirect/drawIndexedIndirect.
 
-TODO: plan and implement
-- indirect draws:
-    - indirectBuffer is {valid, invalid, destroyed, doesn't have usage)
-    - indirectOffset is {
-        - 0, 1, 4
-        - b.size - sizeof(args struct)
-        - b.size - sizeof(args struct) + min alignment (1 or 2 or 4)
```

Comment:
this case would be OOB, but I think that would go in a different suite?

Reply:
I put it in validation/encoding/cmds/render/other. There's similar stuff there for setIndexBuffer